### DISCUSSION TITLE
My Contrubitions

### DISCUSSION THREAD

**Discussion by yahyaerdoan** [community/community#114607] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My Contrubitions not working. Who knows the solution? Can you help me

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @yahyaerdoan ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.


This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
